src/components/AllCards/Card/__snapshots__/Card.test.js.snap

```diff
@@ -0,0 +1,236 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Card /> should render component witout crash 1`] = `
+ShallowWrapper {
```

Comment:
These snapshots are not very readable. we can fix this by using https://github.com/rogeliog/jest-serializer-enzyme

Reply:
I install https://github.com/adriantoine/enzyme-to-json, because I have a problem and I find this issue https://github.com/rogeliog/jest-serializer-enzyme/issues/15#issuecomment-332925891